apps/condo/domains/ticket/components/TicketId/TicketFileList.tsx

```diff
@@ -49,8 +53,30 @@ export const TicketFileList: React.FC<ITicketFileListProps> = ({ files }) => {
     })), [files])
 
     const downloadFile = useCallback(async (file: UploadFile) => {
-        const response = await fetch(file.url)
-        const blob = await response.blob()
+        /*
+        * NOTE
+        * Problem:
+        *   In the case of a redirect according to the scheme: A --request--> B --redirect--> C,
+        *   it is impossible to read the response to the request.
+        *
+        * Solution:
+        *   When adding the "shallow-redirect" header,
+        *   the redirect link to the file comes in json format and a second request is made to get the file.
+        *   Thus, the scheme now looks like this: A --request(1)--> B + A --request(2)--> C
+        * */
+        const redirectResponse = await fetch(file.url, {
+            credentials: 'include',
+            headers: {
+                'shallow-redirect': 'true',
+            },
+        })
+        if (!redirectResponse.ok) throw new Error(ERROR_DOWNLOAD_FILE)
+        const json = await redirectResponse.json()
+        const redirectUrl = json.redirectUrl
+        const fileResponse = await fetch(redirectUrl)
+        if (!fileResponse.ok) throw new Error(ERROR_DOWNLOAD_FILE)
+
+        const blob = await fileResponse.blob()
         const blobUrl = window.URL.createObjectURL(blob)
         const a = document.createElement('a')
```

Comment:
do you want to add `_blank`?

Reply:
`_blank` in function handleFileDownload, line 97. why is it here?